openedx/features/learner_profile/static/learner_profile/js/views/learner_profile_fields.js

```diff
@@ -57,6 +62,10 @@
                     this.$('.u-field-value select').val('private');
                     this.disableField(true);
                 }
+            },
+
+            showSaveButton: function() {
+                $('.btn-change-privacy').css('display', 'inline-block');
```

Comment:
Can we achieve this thing by adding/removing class instead of direct adding CSS.